tutorials/stats/2_pipeline_integration.py

```diff
@@ -0,0 +1,107 @@
+# %% [markdown]
+"""
+# 2. Pipeline Integration
+
+In the DFF ecosystem, extractor functions act as regular extra handlers (see `pipeline` module
+[documentation](https://deeppavlov.github.io/dialog_flow_framework/tutorials/
+tutorials.pipeline.7_extra_handlers_basic.html) ).
```

Comment:
The new line is treated as a whitespace and is invalidating the link. We should either wait for #195 or escape the newline:
https://stackoverflow.com/a/27071588.

Reply:
Done